.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Workload.ManagedGame"
+  ]
+}
```

Comment:
this is maybe some visual studio/code internal file?
probably should be ignored then

Reply:
done